Update policy-control-diagnostic-data-mobile.md (#7401)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/policy-control-diagnostic-data-mobile.md

- **hide** - Hide chat.
- **hideChannel** - Hide a channel from the teams and channel list.
- **image** - Image.

> - **inAppNotification**- Triggered when notification is tapped while user is active in the app

Reviewer: ```suggestion
- **inAppNotification**- Triggered when a notification is tapped while the user is active in the app.
```

- **notBlockedDevice** - A user doesn't reach the threshold for background activity failures in 30 days.
- **notNow** - **Not now** is selected in reminder.
- **notNowUpdate** - UpdateDefer.

> - **notification/ notification_clicked** – Triggered when notification is tapped

Reviewer: ```suggestion
- **notification/ notification_clicked** – Triggered when a notification is tapped.
```

- **showCard** - Tap on card buttons. Cards are key platform constructs and measuring their usage and pattern is necessary to understand platform usage and keep a look out for potential issues on the client side.
- **shownReadReceiptNotice** - The user shown feature notice with settings options.
- **signIn** - **Sign in** is selected on welcome page, or the **Sign In** button is tapped.

> - **SignInWithOTP** - User selects the option to sign in as a guest with one time passcode (OTP). 

Reviewer: ```suggestion
- **SignInWithOTP** - User selected the option to sign in as a guest with one time passcode (OTP). 
```

- **signUp** - **Create a free account** or **Sign up for free** is selected.

> - **SignUpFromSignIn**- User taps on **Create a new account** option from sign-in

Reviewer: ```suggestion
- **SignUpFromSignIn**- User taps on **Create a new account** option from sign-in.
```

- **simultaneousCallForward** - Triggered when:
  - Simultaneous call forward target is set.
  - Simultaneous call forwarding is enabled (Calls ring me is enabled & Also ring is set).

> - **skipVerificationForLink** - Thw user chose to skip verification.

Reviewer: ```suggestion
- **skipVerificationForLink** - The user chose to skip verification.
```

- **smartReply** - Smart reply toggle button is clicked.
- **SMSSendMessage** - The user sends a SMS message.
- **sortChanged** - Triggers when user changes sort order while viewing a tasklist.

> - **SSOAccountListItem**: Triggered when user taps on an SSO account to sign-in

Reviewer: ```suggestion
- **SSOAccountListItem**: Triggered when the user taps on an SSO account to sign in.
```

- **stuckOnConnectingRetrySelected** - **Retry** is selected on the drawer.
- **stuckOnConnectingShownDismissed** - A user dismissed the drawer.
- **suggested_place_selected** - A user shares a static location by selecting a suggested place.

> - **Switching**- Tenant or account is switched from the app. This is required to measure account/ tenant switch issues proactively and provide smoot account/ tenant switch experience.

Reviewer: ```suggestion
- **Switching**- Tenant or account is switched from the app. This is required to measure account/tenant switch issues proactively and provides a smooth account/tenant switch experience.
```

- **meetingFiles** - Triggered when meeting files screen is opened.
- **meetNowActionSheet** - Triggered when user creates a Meet Now meeting.
- **navPersonalFiles** - Triggered when navigation to files screen is performed.

> - **signInSSOPage**: Triggered when user views single sign on page while sign in

Reviewer: ```suggestion
- **signInSSOPage**: Triggered when the user views a single sign-on page while signed in.
```

> -- **signInError**: Triggered when user hits any error while sign in. This is needed to proactively identify and fix issues that users face during sign in. 

Reviewer: ```suggestion
-- **signInError**: Triggered when the user hits any error while signed in. This is needed to proactively identify and fix issues that users face during sign-in. 
```

> -- **TfLSignInSuccessful** : Triggered when user successfully signs in to personal Microsoft account. This is needed to understand sign in and sign-up reliability and proactively identify and fix issues.

Reviewer: ```suggestion
-- **TfLSignInSuccessful**: Triggered when the user successfully signs in to a personal Microsoft account. This is needed to understand sign-in and sign-up reliability and proactively identify and fix issues.
```

> -- **TfWFreemiumSignInSuccessful** : Triggered user successfully signs in to freemium account. This is needed to understand sign in and sign-up reliability and proactively identify and fix issues.

Reviewer: ```suggestion
-- **TfWFreemiumSignInSuccessful**: Triggered when the user successfully signs in to a freemium account. This is needed to understand sign-in and sign-up reliability and proactively identify and fix issues.
```

> -- **TfWSignInSuccessful** : Triggered when user successfully signs in to work or school account. This is needed to understand sign in and sign-up reliability and proactively identify and fix issues.

Reviewer: ```suggestion
-- **TfWSignInSuccessful**: Triggered when the user successfully signs in to a work or school account. This is needed to understand sign-in and sign-up reliability and proactively identify and fix issues.
```

### Scenario

> [!NOTE]
> For information on the properties of PanelAction events, see [Properties sent with scenario events](#properties-sent-with-scenario-events).
> 

> -- **acquire_resource_token_interactive**- Required service call which is triggered when authentication token is acquired by interactive sign in. 

Reviewer: ```suggestion
-- **acquire_resource_token_interactive**- Required service call which is triggered when an authentication token is acquired by interactive sign-in. 
```

> -- **acquire_resource_token_silent**- Required service call which is triggered when authentication token is acquired by silent sign in.

Reviewer: ```suggestion
-- **acquire_resource_token_silent**- Required service call which is triggered when an authentication token is acquired by silent sign-in.
```

> -- **app_crash2** – Triggered when app is crashed unexpectedly. Provides information on how frequently Teams app is crashing. 

Reviewer: ```suggestion
-- **app_crash2** – Triggered when the app is crashed unexpectedly. Provides information on how frequently the Teams app is crashing. 
```

- **app_incremental_sync_launch** Confirms that the pill count gets updated successfully for cold launch.
- **app_incremental_sync_resume** Confirms that the pill count gets updated successfully for warm/hot launch.
- **app_start_cold** To monitor cold app launch (Android only).
- **app_start_hot** To monitor hot app launch (Android only).
- **app_start_warm** To monitor warm app launch (Android only).

> - **auth_adal_tokens**- Required service call to do silent authentication. Triggered when user starts the app, or token is refreshed on expiry.

Reviewer: ```suggestion
- **auth_adal_tokens**- Required service call to do silent authentication. Triggered when a user starts the app, or the token is refreshed on expiry.
```

- **chat_add_giphy** - Confirms that the Giphy GIF rendering action succeeded or failed.

> - **chat_send_message_sfc**- Triggered when chat message is sent in Sfc interop chat

Reviewer: ```suggestion
- **chat_send_message_sfc**- Triggered when a chat message is sent in SfC interop chat.
```

> - **cortanaError** To monitor Cortana error happens.

Reviewer: ```suggestion
- **cortanaError** - To monitor Cortana error happens.
```

- **cortanaView** - To monitor Cortana canvas appear.

> - **cortanaRestart** To monitor Cortana restart.

Reviewer: ```suggestion
- **cortanaRestart** - To monitor Cortana restart.
```

- **cortana_skill_action_delay** - Confirms the start of delay action.
- **cortana_watchdog** - To monitor Cortana watchdog recovery process.
- **create_default_plan_and_nav_to_view** - Confirms successful creation of a default shared task list and how long it took for a user to land on the resulting view after action.

> - **create_new_chat_thread_sfc**- Triggered when new chat thread is created for sfc interop chat

Reviewer: ```suggestion
- **create_new_chat_thread_sfc**- Triggered when a new chat thread is created for an SfC interop chat.
```

- **delete_personal_task** - Confirms the successful deletion of a personal task item.
- **delete_planner_plan** - Confirms the successful deletion of a shared task list.
- **delete_planner_task** - Confirms the successful deletion of a shared task item.
- **json_parse_failure**- Provides information on the frequently of JSON parsing issues.
- **getProfilePicture**- Necessary service call to get user profile picture. 

> - **get_resource_token_async**: Required service call to acquires tokens for Azure Active Directory resources asynchronously
> - **get_resource_token_sync**: Required service call to acquires tokens for Azure Active Directory resources synchronously

Reviewer: ```suggestion
- **get_resource_token_async**: Required service call to acquire tokens for Azure Active Directory resources asynchronously.
- **get_resource_token_sync**: Required service call to acquires tokens for Azure Active Directory resources synchronously.
```

> - **get_sender_sub_scenario** - get sender sub scenario in activity.
> - **interactiveAuthNopa2** – Triggered when no password user is interrupted to do interactive authentication

Reviewer: ```suggestion
- **get_sender_sub_scenario** - Get sender sub scenario in activity.
- **interactiveAuthNopa2** – Triggered when no password user is interrupted to do interactive authentication.
```

- **rename_personal_plan** - Confirms the successful renaming of a personal task list.
- **rename_planner_plan** - Confirms the successful renaming of a shared task list.

> - **save_image** Confirms that the image save action succeeded or failed.
> - **saveMeProfile**- Required service call that gets triggered when user saves the profile

Reviewer: ```suggestion
- **save_image** - Confirms that the image save action succeeded or failed.
- **saveMeProfile**- Required service call that gets triggered when the user saves the profile.
```

> - **share_image** Confirms that the image share action succeeded or failed.

Reviewer: ```suggestion
- **share_image** - Confirms that the image share action succeeded or failed.
```

> - **meeting_details** Confirms that the meeting details sync has succeeded or failed.
> - **show_meeting_participants** Confirms that showing the meeting participant list has succeeded or failed.
> - **search** Confirms that the whole search session has succeeded or failed.
> - **time_based_retention_shared_channel** – Captures performance data for pruning the database

Reviewer: ```suggestion
- **meeting_details** - Confirms that the meeting details sync has succeeded or failed.
- **show_meeting_participants** - Confirms that showing the meeting participant list has succeeded or failed.
- **search** - Confirms that the whole search session has succeeded or failed.
- **time_based_retention_shared_channel** – Captures performance data for pruning the database.
```
